samples/SocketsSample/.gitignore

```diff
@@ -0,0 +1 @@
+wwwroot/js
```

Comment:
OK. as per my comment below I can unify where we put signalr (e.g. wwwroot/lib/signalr) and ignore this in the main .gitignore.